src/styles/themes/default.js

```diff
@@ -48,4 +48,5 @@ export default {
     placeholderText: colors.gray3,
     heroCard: colors.blue,
     uploadPreviewActivityIndicator: colors.gray1,
+    overlayBackgroundColor: colors.translucentWhite,
```

Comment:
Similar to how we are a bit more specific for `modalBackground` - perhaps we do something similar and call this `dragAndDropBackground`

Reply:
I just realized I contradicted this in a later review. @shawnborton maybe you can provide some insight on why you prefer the more specific theme color here?

Reply:
overlayBackgroundColor just feels a bit too generic. Is that the overlay we use for modals? It's actually not, so maybe we should call it something more specific.